tests/foreman/api/test_capsulecontent.py

```diff
@@ -188,7 +188,7 @@ def test_positive_checksum_sync(
         cvv = cvv.read()
         assert len(cvv.environment) == 2
 
-        # Verify repodata's checksum type is sha256, not sha1 on capsule
+        # Verify repodata's checksum type is sha256, not sha512 on capsule
```

Comment:
@vsedmik should we use `checksum_type` variable so that it's easier to update this test in the future?(non-blocking comment)